Not compiling on Apple Silicon?

I'm trying to use this plugin on a new MacBook Pro, but attempting to build it creates the following errors.

<details>
[00m[37mcargo build --release
   Compiling moveline v0.3.1 (/Users/mu/.local/share/nvim/lazy/moveline.nvim)
[00m[37merror: linking with `cc` failed: exit status: 1
  |
  = note: LC_ALL="C" PATH="/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/bin:/Users/mu/.asdf/shims:/Users/mu/.asdf/bin:/opt/homebrew/opt/libpq/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/mu/.local/share/bob/nvim-bin:/Users/mu/.config/composer/vendor/bin:/Users/mu/.cargo/bin:/Users/mu/.config/phpmon/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Applications/kitty.app/Contents/MacOS" VSLANG="1033" ZERO_AR_DATE="1" "cc" "-Wl,-exported_symbols_list,/var/folders/rb/3dlc977d1xd_5m1fz4sg3q2r0000gn/T/rustcVfJE5b/list" "-arch" "arm64" "/var/folders/rb/3dlc977d1xd_5m1fz4sg3q2r0000gn/T/rustcVfJE5b/symbols.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.0.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.1.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.10.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.11.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.12.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.13.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.14.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.15.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.2.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.3.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.4.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.5.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.6.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.7.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.8.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.moveline.a6a071df-cgu.9.rcgu.o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/moveline.5ebpla62ap45qcsg.rcgu.o" "-L" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps" "-L" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libnvim_utils-0f4c7c255781adb7.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libmlua-ad4d65dfdc145d73.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/liberased_serde-3ff06647e8ad6a77.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libonce_cell-57730b96c617e233.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libserde-356f627cc8456d35.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/librustc_hash-1a57d0e77ed159f4.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libnum_traits-d3fb94962dc12cd7.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libbstr-e555fc09d1c2511e.rlib" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libmemchr-9c617dd52afa1bd9.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libstd-24c9557e4e1ae835.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libpanic_unwind-ed2ffefc856f8a16.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libobject-6f6009c5b8498939.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libmemchr-b75b1ad521af5eea.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libaddr2line-dcc7a7d044975758.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libgimli-804bfa7ccacb595a.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/librustc_demangle-3d59e8f5efb6feb0.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libstd_detect-f3aad7e883eb77ef.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libhashbrown-6ef206f3e9399a48.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libminiz_oxide-61abb86383ed392f.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libadler-0af88482d9e5b133.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_alloc-f1ef4a6be77855bb.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libunwind-7d164e838fd2cb9d.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libcfg_if-b87ac2232e8abbd5.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/liblibc-42cfb70398518024.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/liballoc-fad0d1d0f22569c0.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/librustc_std_workspace_core-878cd7c315827c50.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libcore-a1feb39435bcda1f.rlib" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib/libcompiler_builtins-b88cb1ca5c89d224.rlib" "-lSystem" "-lc" "-lm" "-L" "/opt/homebrew/Cellar/rust/1.68.0/lib/rustlib/aarch64-apple-darwin/lib" "-o" "/Users/mu/.local/share/nvim/lazy/moveline.nvim/target/release/deps/libmoveline.dylib" "-Wl,-dead_strip" "-dynamiclib" "-Wl,-dylib" "-nodefaultlibs"
  = note: Undefined symbols for architecture arm64:
            "_luaL_callmeta", referenced from:
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_luaL_error", referenced from:
                mlua::ffi::lua51::compat::lua_rotate::h3055fe7577e6faa7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::init_from_ptr::_$u7b$$u7b$closure$u7d$$u7d$::do_call::h756e5e37aa1f04ad in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h6bed969577b35de9 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::callback_error::h58cbb43cb774e7e6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_luaL_getmetafield", referenced from:
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_luaL_ref", referenced from:
                mlua::util::protect_lua_closure::do_call::h66fdef5fb1377cac (.llvm.8533608232728471067) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_checkstack", referenced from:
                mlua::lua::Lua::entrypoint1::haec3ce6b8510fa22 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_string::hb1c407df0324dbc1 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_sequence_from::hdecc049501afc2a6 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::table::Table::get::h6d8dd5b4ccca26fd in moveline.moveline.a6a071df-cgu.6.rcgu.o
                _$LT$mlua..table..Table$u20$as$u20$mlua..table..TableExt$GT$::call_function::h05fe1199db626ba9 in moveline.moveline.a6a071df-cgu.6.rcgu.o
                mlua::function::Function::call::h1911d97fb08001df in moveline.moveline.a6a071df-cgu.8.rcgu.o
                mlua::function::Function::call::hec9652d7ca2e2763 in moveline.moveline.a6a071df-cgu.8.rcgu.o
                ...
            "_lua_concat", referenced from:
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_createtable", referenced from:
                mlua::util::protect_lua_closure::do_call::h23f3a08acc943bae (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::create_sequence_from::hdecc049501afc2a6 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_table_with_capacity::h2c138710fb57683a in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_closure::do_call::ha4d5b3489c1d4038 (.llvm.8533608232728471067) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::serde::init_metatables::do_call::h80e465dea510dc6c (.llvm.4678935108306754572) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.12.rcgu.o)
                mlua::lua::Lua::create_sequence_from::h0aadd618888f93c3 in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.7.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h1ca9d005ea19e25c (.llvm.18416074522899014774) in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.8.rcgu.o)
                ...
            "_lua_error", referenced from:
                mlua::lua::Lua::entrypoint1::haec3ce6b8510fa22 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::callback_error::h58cbb43cb774e7e6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_error_registry::error_tostring::h6a62528aa528db7d in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_error_registry::destructed_error::h37c8f19f6d3d27dd in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_getinfo", referenced from:
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_getmetatable", referenced from:
                mlua::table::Table::get::h6d8dd5b4ccca26fd in moveline.moveline.a6a071df-cgu.6.rcgu.o
                _$LT$mlua..table..Table$u20$as$u20$mlua..table..TableExt$GT$::call_function::h05fe1199db626ba9 in moveline.moveline.a6a071df-cgu.6.rcgu.o
                mlua::util::pop_error::h7861d72b649d8805 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::get_gc_userdata::hd2e47c7775f5e1f8 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::error_traceback::hd08fa7f86f646add in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                _$LT$core..panic..unwind_safe..AssertUnwindSafe$LT$F$GT$$u20$as$u20$core..ops..function..FnOnce$LT$$LP$$RP$$GT$$GT$::call_once::h59ed6e5b1ecfb488 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.14.rcgu.o)
                mlua::table::Table::get::h435510da479c51e5 in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.2.rcgu.o)
                ...
            "_lua_getstack", referenced from:
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_gettable", referenced from:
                mlua::table::Table::get::do_call::he9da81b0fa39e83e in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.4.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h31e569ba5737b09f (.llvm.18416074522899014774) in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.8.rcgu.o)
            "_lua_gettop", referenced from:
                mlua::util::protect_lua_closure::h3b64f9d453db2858 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::h3ccf02c40f8eb2e9 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::hb30030bc02168a24 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::h23f3a08acc943bae (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::hc3473f99f3d74545 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::hfe7cb9a650a32a02 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::entrypoint1::haec3ce6b8510fa22 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                ...
            "_lua_insert", referenced from:
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::lua_rotate::h3055fe7577e6faa7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::init_from_ptr::_$u7b$$u7b$closure$u7d$$u7d$::do_call::h756e5e37aa1f04ad in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_call::h7830d7eb69b8beac in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::protect_lua_closure::h32e5bfa196672736 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::protect_lua_closure::h4ebd0390e16c30f7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_isnumber", referenced from:
                mlua::lua::Lua::coerce_integer::h2115e994a3abf0f9 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::coerce_number::h3b127f9cbb4667cb in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::to_string::hee6ffd8c138b5462 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_isstring", referenced from:
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_newthread", referenced from:
                mlua::util::protect_lua_closure::do_call::h66fdef5fb1377cac (.llvm.8533608232728471067) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_newuserdata", referenced from:
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h1630b46edf5c920f in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h19bfa7e82f33409f in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h6c8ee9d4a27bd0c7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::habff8f45777ae38c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hcf4e1d6bf1ebf717 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hd96a73dd91007b81 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_next", referenced from:
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_pcall", referenced from:
                mlua::util::protect_lua_closure::h3b64f9d453db2858 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::h3ccf02c40f8eb2e9 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::hb30030bc02168a24 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::function::Function::call::h1911d97fb08001df in moveline.moveline.a6a071df-cgu.8.rcgu.o
                mlua::function::Function::call::hec9652d7ca2e2763 in moveline.moveline.a6a071df-cgu.8.rcgu.o
                mlua::util::protect_lua_call::h7830d7eb69b8beac in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::protect_lua_closure::h32e5bfa196672736 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_pushboolean", referenced from:
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::init_gc_metatable::h44356c4ba0e63e22 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_gc_metatable::hd51969f2cf304bd9 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_gc_metatable::hfb20e433cef0c9cd in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_error_registry::h8f96db6569f4b5b3 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::serde::init_metatables::do_call::h80e465dea510dc6c (.llvm.4678935108306754572) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.12.rcgu.o)
            "_lua_pushcclosure", referenced from:
                mlua::util::protect_lua_closure::h3b64f9d453db2858 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::h3ccf02c40f8eb2e9 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::hb30030bc02168a24 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::function::Function::call::h1911d97fb08001df in moveline.moveline.a6a071df-cgu.8.rcgu.o
                mlua::function::Function::call::hec9652d7ca2e2763 in moveline.moveline.a6a071df-cgu.8.rcgu.o
                mlua::lua::Lua::create_callback::hfefc69134b38c716 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::create_callback::do_call::h97e74d1f9690481a in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
            "_lua_pushfstring", referenced from:
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_pushinteger", referenced from:
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h31e569ba5737b09f (.llvm.18416074522899014774) in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.8.rcgu.o)
            "_lua_pushlightuserdata", referenced from:
                mlua::util::protect_lua_closure::h3b64f9d453db2858 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::h3ccf02c40f8eb2e9 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::hb30030bc02168a24 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::init_from_ptr::h7c7de2befb5a8c60 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::make_from_ptr::h238ff6b6282ab687 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::init_from_ptr::_$u7b$$u7b$closure$u7d$$u7d$::do_call::h756e5e37aa1f04ad in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
            "_lua_pushlstring", referenced from:
                mlua::util::protect_lua_closure::do_call::hfe7cb9a650a32a02 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::create_string::hb1c407df0324dbc1 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_sequence_from::hdecc049501afc2a6 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h0e1b4d41ee9a236d (.llvm.8533608232728471067) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_pushnil", referenced from:
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                core::ptr::drop_in_place$LT$mlua..table..Table$GT$::hae1508631e830b8d in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                core::ptr::drop_in_place$LT$mlua..value..Value$GT$::hb79051b10e1db0b4 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::callback_error_ext::_$u7b$$u7b$closure$u7d$$u7d$::ha0881786aec0951e in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                core::ptr::drop_in_place$LT$mlua..value..Value$GT$::hb79051b10e1db0b4 (.llvm.17378053517906720843) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.8.rcgu.o)
                ...
            "_lua_pushnumber", referenced from:
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
            "_lua_pushstring", referenced from:
                mlua::serde::init_metatables::do_call::h80e465dea510dc6c (.llvm.4678935108306754572) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.12.rcgu.o)
            "_lua_pushthread", referenced from:
                mlua::lua::Lua::init_from_ptr::h7c7de2befb5a8c60 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
            "_lua_pushvalue", referenced from:
                mlua::ffi::lua51::compat::lua_rotate::h3055fe7577e6faa7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::globals::hfcf638d757cfcde3 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_ref::h5e6994f68e38d8f7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::callback_error_ext::_$u7b$$u7b$closure$u7d$$u7d$::ha0881786aec0951e in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
            "_lua_rawequal", referenced from:
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::util::pop_error::h7861d72b649d8805 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::get_gc_userdata::hd2e47c7775f5e1f8 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::error_traceback::hd08fa7f86f646add in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                _$LT$core..panic..unwind_safe..AssertUnwindSafe$LT$F$GT$$u20$as$u20$core..ops..function..FnOnce$LT$$LP$$RP$$GT$$GT$::call_once::h59ed6e5b1ecfb488 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.14.rcgu.o)
            "_lua_rawget", referenced from:
                mlua::table::Table::get::h6d8dd5b4ccca26fd in moveline.moveline.a6a071df-cgu.6.rcgu.o
                _$LT$mlua..table..Table$u20$as$u20$mlua..table..TableExt$GT$::call_function::h05fe1199db626ba9 in moveline.moveline.a6a071df-cgu.6.rcgu.o
                mlua::lua::Lua::init_from_ptr::h7c7de2befb5a8c60 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::make_from_ptr::h238ff6b6282ab687 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::push_gc_userdata::habff8f45777ae38c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hc5bf2b0b06337cc7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hcf4e1d6bf1ebf717 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_rawgeti", referenced from:
                _$LT$mlua..table..TableSequence$LT$V$GT$$u20$as$u20$core..iter..traits..iterator..Iterator$GT$::next::h64dc15dbc5bbb5ef in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.2.rcgu.o)
            "_lua_rawset", referenced from:
                mlua::lua::Lua::init_from_ptr::_$u7b$$u7b$closure$u7d$$u7d$::do_call::h756e5e37aa1f04ad in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h6295b163e241a840 (.llvm.8533608232728471067) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h6bed969577b35de9 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_error_registry::do_call::h93483f0735513713 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::init_error_registry::do_call::hb1da94f60feaf0c0 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::table::Table::raw_set::do_call::he050c432d125ffbc in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.4.rcgu.o)
                mlua::serde::init_metatables::do_call::h80e465dea510dc6c (.llvm.4678935108306754572) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.12.rcgu.o)
                ...
            "_lua_rawseti", referenced from:
                mlua::util::protect_lua_closure::do_call::hc3473f99f3d74545 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::create_sequence_from::hdecc049501afc2a6 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_sequence_from::h0aadd618888f93c3 in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.7.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h70007f54589608b5 (.llvm.18416074522899014774) in libnvim_utils-0f4c7c255781adb7.rlib(nvim_utils-0f4c7c255781adb7.nvim_utils.82d1585f-cgu.8.rcgu.o)
            "_lua_remove", referenced from:
                mlua::util::protect_lua_closure::h3b64f9d453db2858 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::h3ccf02c40f8eb2e9 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::hb30030bc02168a24 in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_call::h7830d7eb69b8beac in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_replace", referenced from:
                mlua::ffi::lua51::compat::lua_rotate::h3055fe7577e6faa7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                core::ptr::drop_in_place$LT$mlua..table..Table$GT$::hae1508631e830b8d in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                core::ptr::drop_in_place$LT$mlua..value..Value$GT$::hb79051b10e1db0b4 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
            "_lua_setmetatable", referenced from:
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::push_gc_userdata::habff8f45777ae38c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hc5bf2b0b06337cc7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hcf4e1d6bf1ebf717 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::push_gc_userdata::hd96a73dd91007b81 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::userdata_destructor::h206c2fc39d93f45f in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::userdata_destructor::h44d15214401312b5 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_settable", referenced from:
                mlua::table::Table::set::do_call::hda874fd4b5e588ca in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.4.rcgu.o)
            "_lua_settop", referenced from:
                mlua::util::protect_lua_closure::do_call::h23f3a08acc943bae (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::hc3473f99f3d74545 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::hfe7cb9a650a32a02 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::table::Table::get::h6d8dd5b4ccca26fd in moveline.moveline.a6a071df-cgu.6.rcgu.o
                _$LT$mlua..table..Table$u20$as$u20$mlua..table..TableExt$GT$::call_function::h05fe1199db626ba9 in moveline.moveline.a6a071df-cgu.6.rcgu.o
                mlua::function::Function::call::h1911d97fb08001df in moveline.moveline.a6a071df-cgu.8.rcgu.o
                mlua::function::Function::call::hec9652d7ca2e2763 in moveline.moveline.a6a071df-cgu.8.rcgu.o
                ...
            "_lua_toboolean", referenced from:
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::pop_value::h0046a76d872f4d86 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::to_string::hee6ffd8c138b5462 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_tolstring", referenced from:
                mlua::util::protect_lua_closure::do_call::hfe7cb9a650a32a02 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::create_string::hb1c407df0324dbc1 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::lua::Lua::create_sequence_from::hdecc049501afc2a6 in moveline.moveline.a6a071df-cgu.4.rcgu.o
                mlua::ffi::lua51::compat::luaL_traceback::h323be3ace596c448 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::coerce_string::h643caf411ac7becc in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::protect_lua_closure::do_call::h0e1b4d41ee9a236d (.llvm.8533608232728471067) in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                ...
            "_lua_tonumber", referenced from:
                mlua::lua::Lua::coerce_integer::h2115e994a3abf0f9 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::coerce_number::h3b127f9cbb4667cb in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::pop_value::h0046a76d872f4d86 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::to_string::hee6ffd8c138b5462 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_topointer", referenced from:
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::init_from_ptr::h7c7de2befb5a8c60 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::util::get_gc_userdata::hd2e47c7775f5e1f8 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
                mlua::util::to_string::hee6ffd8c138b5462 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.0.rcgu.o)
            "_lua_touserdata", referenced from:
                mlua::util::protect_lua_closure::do_call::h23f3a08acc943bae (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::hc3473f99f3d74545 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::util::protect_lua_closure::do_call::hfe7cb9a650a32a02 (.llvm.7221355180800964757) in moveline.moveline.a6a071df-cgu.10.rcgu.o
                mlua::lua::Lua::pop_value::h0046a76d872f4d86 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::make_from_ptr::h238ff6b6282ab687 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::callback_error_ext::_$u7b$$u7b$closure$u7d$$u7d$::ha0881786aec0951e in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
            "_lua_type", referenced from:
                mlua::table::Table::get::h6d8dd5b4ccca26fd in moveline.moveline.a6a071df-cgu.6.rcgu.o
                _$LT$mlua..table..Table$u20$as$u20$mlua..table..TableExt$GT$::call_function::h05fe1199db626ba9 in moveline.moveline.a6a071df-cgu.6.rcgu.o
                mlua::ffi::lua51::compat::compat53_findfield::h32aad1ee10244553 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
                mlua::lua::Lua::init_from_ptr::h7c7de2befb5a8c60 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::pop_value::h0046a76d872f4d86 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::create_callback::call_callback::h48057cac48c68d94 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
            "_lua_typename", referenced from:
                mlua::ffi::lua51::compat::luaL_tolstring::h3ddf9680efaec4f6 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.5.rcgu.o)
            "_lua_xmove", referenced from:
                mlua::lua::Lua::create_table_with_capacity::h2c138710fb57683a in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::globals::hfcf638d757cfcde3 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::coerce_string::h643caf411ac7becc in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_value::had80e30cf933342c in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::pop_value::h0046a76d872f4d86 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::push_ref::h5e6994f68e38d8f7 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                mlua::lua::Lua::pop_ref::h7632f80898d67013 in libmlua-ad4d65dfdc145d73.rlib(mlua-ad4d65dfdc145d73.mlua.d1891646-cgu.1.rcgu.o)
                ...
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

[00m[37merror: could not compile `moveline` due to previous error
[00m[37mmake: *** [build] Error 101
[00m
</details>

Any ideas how to fix something like this?

--- a/.cargo/config.toml
+++ b/.cargo/config.toml
@@ -1,0 +1,5 @@
+[target.x86_64-apple-darwin]
+rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]
+
+[target.aarch64-apple-darwin]
+rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,9 +14,3 @@ crate-type = ["cdylib"]
 [dependencies]
 nvim-utils = { version = "0.1.12" }
 mlua = { version = "0.8.7", features = ["luajit", "vendored", "module"] }
-
-[target.x86_64-apple-darwin]
-rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]
-
-[target.aarch64-apple-darwin]
-rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]

